src/execution/operator/join/physical_hash_join.cpp

```diff
@@ -733,9 +733,9 @@ void JoinFilterPushdownInfo::PushInFilter(const JoinFilterPushdownFilter &info,
 	return;
 }
 
-unique_ptr<DataChunk> JoinFilterPushdownInfo::Finalize(ClientContext &context, JoinHashTable &ht,
+unique_ptr<DataChunk> JoinFilterPushdownInfo::Finalize(ClientContext &context, JoinHashTable *ht,
```

Comment:
Could this be an `optional_ptr`?

Reply:
Yes it could! Sorry old C hacker here and I forget sometimes.  Mark has already merged this but here will be more PRs as I add this to other inequality join operators and I will fix it then.